03-plugins.md

```diff
@@ -71,7 +70,7 @@ $(document).ready(function() {
   $('#summernote').summernote({
     popover: {
       image: [
-        // This is a Custom Button in a new Toolbar Area
+// This is a Custom Button in a new Toolbar Area
```

Comment:
I thought you missed this indentation.

Reply:
Yep, looks like I did, I'll check again, ooopppsss